gooey/gui/components/footer.py

```diff
@@ -122,7 +122,7 @@ def button(self, label=None, style=None, event_id=-1):
         return wx.Button(
             parent=self,
             id=event_id,
-            size=(90, 24),
+            size=(90, 30),

```

Comment:
Hm.. I'm not sure why I hard coded that height to begin with.. but if you set it to `-1` it should pick an appropriate height for all platforms (e.g. `size=(90, -1)`).